misc/dev-support/docker/infrastructure/env-files/soft_panda_uat.env

```diff
@@ -17,6 +17,8 @@ BRANCH_NAME=soft_panda_uat
 # temporary change, to apply migration-scripts against the DB from gh-actions
 #DB_PORT=25432
 DB_PORT=32432
+DB_DOCKER_IMG=metasfresh/metas-db:5.175-release-5.175
+#DB_DOCKER_IMG=docker.io/metasfresh/metas-db:5.175-new-dawn-uat-preloaded
```

Comment:
@metas-ts is this change right or is just temporary/wrong commit?

Reply:
imo it's right